components/experimental/src/dimension/provider/currency_compact.rs

```diff
@@ -46,56 +44,35 @@ pub struct ShortCurrencyCompactV1<'data> {
     databake(path = icu_experimental::dimension::provider::currency_compact)
 )]
 #[repr(u8)]
-pub enum CompactCount {
+pub enum Count {
```

Comment:
can you use an existing `Count`? This type exists at least twice already.

Reply:
done and https://github.com/unicode-org/icu4x/pull/5373

Reply:
you just moved it?